onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2260,19 +2260,107 @@ def aten_embedding_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::embedding_bag", trace_only=True)
 def aten_embedding_bag(
-    weight: TensorType,
-    indices: TensorType,
-    offsets: TensorType,
-    scale_grad_by_freq: bool = False,
-    mode: int = 0,
-    sparse: bool = False,
-    per_sample_weights: Optional[TensorType] = None,
+    weight: TFloat,
+    indices: INT64,
+    offsets: INT64 = None,  # Could be None accotding to the doc, go 2d branch
+    scale_grad_by_freq: bool = False,  # pylint: disable=unused-argument
+    mode: int = 0,  # [0,1,2] indicate ["mean", "sum", "max"]
+    sparse: bool = False,  # pylint: disable=unused-argument
+    per_sample_weights: Optional[TFloat] = None,
     include_last_offset: bool = False,
-) -> tuple[TensorType, TensorType, TensorType, TensorType]:
+) -> TFloat:
     """embedding_bag(Tensor weight, Tensor indices, Tensor offsets, bool scale_grad_by_freq=False, int mode=0, bool sparse=False, Tensor? per_sample_weights=None, bool include_last_offset=False) -> (Tensor, Tensor, Tensor, Tensor)"""
```

Comment:
@BowenBao The function returns four outputs. Do you know if they are all used?

Reply:
Looks like unused. Would be best to e2e test it out w/ exporter.

https://github.com/pytorch/pytorch/blob/26e3b4020f01d4fc2b7f63e1de4c94d2c8b362b5/torch/nn/functional.py#L2422

https://github.com/pytorch/pytorch/blob/26e3b4020f01d4fc2b7f63e1de4c94d2c8b362b5/aten/src/ATen/native/EmbeddingBag.cpp#L1233